### PR TITLE
Support  opentracing  v1.0.0-beta6

### PR DESCRIPTION
**Pull request by bnubobby**
it's time to update this library

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,13 +25,13 @@
     "require": {
         "php": "^7.0",
         "ext-sockets": "*",
-        "opentracing/opentracing": "1.0.0-beta5",
+        "opentracing/opentracing": "1.0.0-beta6",
```
Comment (sergeyklay) [inline composer.json]: This will require change php version:
```diff
- "php": "^7.0",
+ "php": "^7.1",
```

Also need to update Travis' build matrix and README.md

File: src/Jaeger/Span.php [hunk 2]
```diff
@@ -396,15 +395,15 @@ public function getLogs(): array
...
-    public function addBaggageItem($key, $value)
+    public function addBaggageItem(string $key, string $value) :void
     {
         $this->context = $this->context->withBaggageItem($key, $value);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getBaggageItem($key)
```
Comment (sergeyklay) [inline src/Jaeger/Span.php]: Travis CI says that there is errors like:
```
PHP Fatal error: 
Declaration of Jaeger\SpanContext::getBaggageItem(string $key): ?string
must be compatible with OpenTracing\SpanContext::getBaggageItem($key)
in /home/travis/build/jonahgeorge/jaeger-client-php/src/Jaeger/SpanContext.php on line 8
```

Comment (bnubobby) [reply src/Jaeger/Span.php]: `
 public function getBaggageItem(string $key): ?string;
`
is right  
[SpanContext.php](https://github.com/opentracing/opentracing-php/blob/master/src/OpenTracing/SpanContext.php)

It seems that Travis CI chose opentracing  of  1.0.0-beta5 ?

Comment (sergeyklay) [reply src/Jaeger/Span.php]: I can't approve/merge this until tests are failed. So, could you please fix you patch?

**Comment by bnubobby** [comment]
Cancel the pull

**Pull request closed without merging**